pallets/gear/src/manager/task.rs

```diff
@@ -89,40 +69,14 @@ where
         let waitlisted = Pallet::<T>::wake_dispatch(program_id, message_id, reason)
             .unwrap_or_else(|| unreachable!("Scheduling logic invalidated!"));
 
+        self.send_signal(message_id, waitlisted.destination());
```

Comment:
I also mentioned one more case for signal in dm: when out of gas reached

Reply:
#1744